brave-jaxrs2/src/main/java/com/github/kristofa/brave/jaxrs2/BraveTraceFeature.java

```diff
@@ -0,0 +1,32 @@
+package com.github.kristofa.brave.jaxrs2;
+
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.http.DefaultSpanNameProvider;
+import com.github.kristofa.brave.http.SpanNameProvider;
+
+public class BraveTraceFeature implements Feature {
+
+    private Brave brave;
+    private SpanNameProvider spanNameProvider;
+
+    public BraveTraceFeature(Brave brave) {
+        this(brave, new DefaultSpanNameProvider());
+    }
+    
+    public BraveTraceFeature(Brave brave, SpanNameProvider spanNameProvider) {
```

Comment:
similar feedback to https://github.com/openzipkin/brave/pull/274#issuecomment-260157403

let's hide the ctor and use a builder. it will make later things easier (such as config for extra tags)